symfony2/yaml/bundles.yml

```diff
@@ -52,8 +52,8 @@ questions:
     -
         question: 'What is the correct load() method definition in Symfony\Component\DependencyInjection\Extension\ExtensionInterface?'
         answers:
-            - {value: "public function load(array $configs, ContainerBuilder $container)", correct: true}
-            - {value: "public function load(array $config, ContainerBuilder $container)",  correct: false}
+            - {value: "public function load(array $configs, ContainerBuilder $container)", correct: false}
+            - {value: "public function load(array $config, ContainerBuilder $container)",  correct: true}
```

Comment:
They are both correct in fact. Only the signature of the method must be the same.

so  `public function load(array $foo, ContainerBuilder $bar)` is also right.